apps/balancer-tools/src/components/Fathom.tsx

```diff
@@ -0,0 +1,39 @@
+"use client";
+
+import { load, trackPageview } from "fathom-client";
+import { usePathname, useSearchParams } from "next/navigation";
+import { Suspense, useEffect } from "react";
+
+const appUrl = "balancer-tools.bleu.fi";
+const fathomId = "HLWGNXQK";
```

Comment:
@ribeirojose not sure it's okay to leave it like this. Thought about adding to an env, but both official docs just say to add the fathom id. So I decided to not over complicate at first, but if necessary change this

Reply:
this is going to be in the frontend anyways, so no worries